plugin/src/queries.ts

```diff
@@ -67,6 +67,75 @@ function bulkOperationQuery(query: string) {
   `;
 }
 
+const collectionsQuery = (dateString?: string) => `
+{
+  collections${
+    dateString
+      ? `(query: "created_at:>='${dateString}' OR updated_at:>='${dateString}'")`
+      : ``
+  } {
+    edges {
+      node {
+        products {
+          edges {
+            node {
+              id
+            }
+          }
+        }
+        description
+        descriptionHtml
+        feedback {
+          details {
+            app {
+              id
+            }
+            link {
+              label
+              url
+            }
+            messages {
+              field
+              message
+            }
+          }
+          summary
+        }
+        handle
+        id
+        image {
+          id
+          altText
+          height
+          width
+          originalSrc
+          transformedSrc
+        }
```

Comment:
Currently none of the swag store collections come back with an image. When I get the chance to test this with some better data, I'll add image processing for this image.